src/agentscope/agents/agent.py

```diff
@@ -40,6 +102,7 @@ def __init__(
         model_config_name: str = None,
         use_memory: bool = True,
         memory_config: Optional[dict] = None,
+        to_dist: Optional[Union[dict, bool]] = False,
```

Comment:
Just `host: str=None` and `port: int=None` is fine (users don't need to search what parameters should be contained here and their format)